### ISSUE TITLE
node --no-deprecation ./dist/src/index.js FATAL: TypeError (vcs dep issue)

### ISSUE THREAD

**Issue by lukepatrick** [Azure/brigade#139]
When running a supplied `brigade.js`, e.g 
```
const { events } = require('brigadier')

events.on("exec", (brigadeEvent, project) => {
  console.log("Hello world!")
})
```

with `brig run -f brigade.js test/project`

Get error:

```
Started brigade-worker-01bz0j243ew0sm61pff1ebsepc-master
yarn run v1.3.2
$ node prestart.js
prestart: src/brigade.js written
$ node --no-deprecation ./dist/src/index.js
FATAL: TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object. (rejection)
error handler is cleaning up
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This assumes the default `values.yaml` that has no vcsSidecar
The docs indicate no vcs assumes GIT? Doesn't seem to apply. If a vcsSidecar is supplied, e.g. `vcsSidecar: "deis/git-sidecar:latest"`
all is good regardless of using a supplied brigade.js

**Comment by bacongobbler** [comment]
I think this was fixed with #136. Would you please check and confirm? :)

**Comment by lukepatrick** [comment]
No change with #136 . Likewise no improvement with the other similar #137
